.github/workflows/openapi-check.yml

```
@@ -47,7 +47,28 @@ jobs:
           # Start server in background to fetch spec from API endpoint
           MIX_ENV=test mix phx.server &
           SERVER_PID=$!
-          sleep 15  # Wait for server to start
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)